.gitignore

```diff
@@ -5,6 +5,7 @@ multidb
 .config
 .yardoc
 .idea
+Gemfile.lock
```

Comment:
Remove the `Gemfile.lock` file from source control.

Each version of Ruby requires slightly different versions of dependencies. Removing the lock allows each CI test step to install dependencies that are compatible with the running version of Ruby.